src/Sulu/Component/Webspace/DefaultTemplate.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\Webspace;
+
+/**
+ * Container for a default template definition.
+ */
+class DefaultTemplate
+{
+    /**
+     * @var string
+     */
+    private $type;
+
+    /**
+     * @var string
+     */
+    private $template;
+
+    /**
+     * @var string|null
+     */
+    private $parentTemplate;
+
+    /**
+     * DefaultTemplate constructor.
+     *
+     * @param string $type
+     * @param string $template
+     * @param string|null $parentTemplate
+     */
+    public function __construct(string $type, string $template, ?string $parentTemplate = null)
+    {
+        $this->type = $type;
+        $this->template = $template;
+        $this->parentTemplate = $parentTemplate;
+    }
+
+    /**
+     * @return string
+     */
+    public function getType(): string
+    {
+        return $this->type;
+    }
+
+    /**
+     * @param string $type
+     */
+    public function setType(string $type): void
```

Comment:
We usually don't implement setter, unless they are really necessary. All the properties in this class are not supposed to be changed after object's construction, therefore I would not have any setters at all.

Reply:
All setters are removed